docs/slack.rst

```diff
@@ -168,6 +168,12 @@ If you still can't find a suitable channel, you can propose a new channel. To pr
 3. Post in **#meta** requesting a channel. Be sure to share your proposed channel name, description, and to tag your cosponsors.
 
 The Write the Docs moderation team will review your proposal.
+If your proposal is approved, then you and your consponsors must:
+
+#. Seed discussion in the new channel.
```

Comment:
```suggestion
#. Make a best-effort to seed and maintain a cadence of conversation.
    For example, you can post questions once a week related to the channel topic and encourage others to participate, similar to how **#learn-tech-writing** contributors have done book clubs.
```

Reply:
I'd like to set the bar to be "If you're going to start this up, you need to do more than show up at the beginning"

Reply:
We've had some questions in the past from folks who were worried they were taking on a permanent commitment by suggesting a channel (e.g., "do I have to moderate the channel, if I propose it?"). I think putting in a requirement like this might scare people off more than it would spur conversation.